ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue

```diff
@@ -462,32 +483,38 @@ export default defineComponent({
 
     const { required, notEqualTo, customRules } = useInputRules()
 
+    const homeLocationInfo: MhrRegistrationHomeLocationIF =
+      props.isTransportPermitReview ? getMhrTransportPermit.value.newLocation : getMhrRegistrationLocation.value
```

Comment:
Why scope this outside of localState?

Reply:
To avoid having `localState.` in front of every `homeLocationInfo` 😃.  I will update, it's not that bad.

Reply:
You may have issues using it in localState because its the source for a lot of the properties. 
Very much feels like the computed value should be the prop vs a flag to toggle between the 2 data sources. 
Your call, can leave as it or update, no biggie.

Reply:
I know your in a crunch, so don't need to stress about this now 👍